### DISCUSSION TITLE
Is there any method to accelerate the computation in Shapely using GPU resources?

### DISCUSSION THREAD

**Discussion by valanceo** [shapely/shapely#2064] [General]
Is there any method to accelerate the computation in Shapely using GPU resources?
Thanks you very much.

**Comment by theroggy** [comment]
No, shapely is a python wrapper around the C++ [GEOS](https://github.com/libgeos/geos) library, and it does not support the use of GPU.

Ref: https://github.com/libgeos/geos/issues/1038
